src/style.css

```diff
@@ -0,0 +1,135 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+.card {
+  width: 200px;
+  background: #fff;
+  border-radius: 5px;
+
+  font-weight: normal;
+  font-size: 11px;
+}
+
+.card__inner-width {
```

Comment:
its looks like a modifier of block "card" if that so, it should be called as "card--inner-width"